lib/collection/form-param.js

```diff
@@ -23,6 +23,7 @@ _.inherit((
         this.value = _.get(options, 'value') || '';
         this.type = _.get(options, 'type');
         this.src = _.get(options, 'src');
+        this.contentType = _.get(options, 'contentType');
```

Comment:
While this is fine for now, can you add a todo before the first `_.get` call to get rid of all these `_.get` calls?

Reply:
Also, shouldn't we validate/sanitize the `contentType` value first?

Reply:
we should accept any standard/custom `content-type`.